src/data/providers.ts

```diff
@@ -48,6 +48,14 @@ export const providersData: Providers = {
                 message: "Install Geyser in the Aternos plugin list and connect to your server with your Java IP and port. See [Aternos's article](https://support.aternos.org/hc/en-us/articles/360051047631) for more details. Do note: Aternos automatically installs the latest version of the plugin, but does not auto update it. To update Geyser on Aternos, simply reinstall the plugin (no need to uninstall the plugin/deleting the plugin files)."
             })
         },
+        {
+            name: 'AxentHost',
+            url: 'https://axenthost.com/games/minecraft/',
+            description: translate({
+                id: 'providers.provider.axenthost.description',
+                message: 'Create a default Minecraft server. On the server control panel, navigate to the "Variants" tab and select "GeyserMC". The server will be automatically reinstalled with Geyser instead of Vanilla.'
```

Comment:
What does this actually install? a paper+geyser server? Or geyser-standalone?
What should users do if they already have a paper / fabric /... server, all of which Geyser supports?

From our end, we created system properties that allow hosts to set port / IP automatically; maybe those can be useful for you:
https://geysermc.org/wiki/geyser/geyser-command-line-arguments-and-system-properties#configuration-system-properties

Reply:
We are based on this GeyserMC config :
https://github.com/GeyserMC/pterodactyl-stuff/blob/master/egg-paper--geyser--floodgate.json

The users don't have to do anything else, they can either use a paper / fabric... server, or use the geyser "variant", which installs a server with the previous config.